library/utils.c

```diff
@@ -289,7 +289,16 @@ static void init_debug(uv_loop_t *loop) {
         tlsuv_set_debug(tlsuv_level, tlsuv_logger);
     }
 
-    ziti_debug_out = stderr;
+    char *logfile = getenv("ZITI_LOG_FILE");
```

Comment:
is there a reason for doing this when the log level is being set? it feels like this belongs in ziti_log_init with explicit API around it.

Reply:
this is just an option for default logger, see comment below